### ISSUE TITLE
[Desktop] Webview is always on top of other components

### ISSUE THREAD

**Issue by JuBan1** [KevinnZou/compose-webview-multiplatform#41]
It's not possible to render composables on top of the WebView on desktop (using JCEF or KCEF. Android/iOS are unaffected). Any composable, even popups or context menus, will be clipped by it.

This isn't a bug with this library but with [Compose's Swing interop](https://github.com/JetBrains/compose-multiplatform/issues/2926).

This issue is to keep track of this bug, and to help others who come looking in search of answers.

**Comment by KevinnZou** [comment]
Thanks for your suggestion! I also noticed that problem and found that it is the problem with the Swing interop. I will also keep watching on this issue. Thanks!

**Comment by MatkovIvan** [comment]
I guess [my PR](https://github.com/JetBrains/compose-multiplatform-core/pull/915) will fix this too

**Comment by alirezat775** [comment]
In this [PR](https://github.com/KevinnZou/compose-webview-multiplatform/pull/120), I updated the compose version to 1.6.0 I guess we can close this issue when the PR merges

**Comment by Kiryushin-Andrey** [comment]
I see that compose version in the library is currently 1.6.1. In my project I'm using CMP version 1.6.1 and compose-webview-multiplatform version 1.9.20 (the latest one), and the issue is still there.  @MatkovIvan @KevinnZou is it expected?

**Comment by MatkovIvan** [comment]
@Kiryushin-Andrey Yes, kinda expected. Rendering interop views above Compose is still default and documented behavior. However, Compose Multiplatform 1.6.0 includes my change (from the comment above) that adds a new experimental mode that allows rendering Compose above platform views. It has some limitations and should be enabled explicitly. See docs or PR description for details.

**Comment by Crazyphil** [comment]
Did someone already manage to draw Composables over a WebView?

I tried following the guide according to the requirements of https://github.com/JetBrains/compose-multiplatform-core/pull/915:
* Windows with Direct3D renderer (this is the default renderer on Windows)
* Offscreen rendering, enabled using `WebViewState.webSettings.desktopWebSettings.offScreenRendering = true`
* Set feature flag system property in the `main` method before initializing the Compose application (`System.setProperty("compose.interop.blending", "true")`)

Executing these three steps, I still can't manage to draw over the WebView (for example, by using `Modifier.border()`).

Does anyone have a minimal working example for drawing over the WebView? Or is this one of the cases where the feature will never work because of a known limitation:
> On DirectX, it cannot overlay another DirectX component

I assume the Chromium engine might use hardware-accelerated drawing using Direct3D as well, which could clash with the requirements for painting over the SwingPanel.

**Comment by Lowae** [comment]
Has there been any progress on this issue? It has been two years, and I've confirmed that the problem is still reproducible using the provided sample in the repository.

**Comment by mtpdog** [comment]
Has anyone managed to do offscreen rendering?

**Comment by mtpdog** [comment]
> I guess [my PR](https://github.com/JetBrains/compose-multiplatform-core/pull/915) will fix this too

Could you please share some example with WebView in the background and Composables on top of it for JVM?

**Comment by Italord0** [comment]
Any progress on this? 

Even adding `System.setProperty("compose.interop.blending", "true"` i'm having the same problem.
